### PR TITLE
HUDs work for monkeys(fixes part of #7025)

### LINKED ISSUE #8699: Medihuds don't work for monkeys
(WEB REPORT BY: archsunder REMOTE: 158.69.55.2:7777)

> Revision 16b9c9b6c966ca8c77c1e458164dcdf081a980e5 
> General description of the issue
> When a monkey wears a medihud, it doesnt show vitals on people or allow access to medical records when you examine people. 
> 
> What you expected to happen

Vital signs

> What actually happened
> 
> Steps to reproduce if possible

### LINKED ISSUE #10829: Health HUD scanner
(WEB REPORT BY: edwarus REMOTE: 172.93.109.202:7777)

> Revision 85c275257b9e840407c9d3258de0168140f4fd34
> 
> General description of the issue
> Health HUD scanner does not work if you are a monkey
> 
> What you expected to happen
> 
> What actually happened
> 
> Steps to reproduce if possible
> turn yourself into a monkey

### PR DESCRIPTION
**Pull request by ghost**
Fixes #8699 too
Fixes #10829 too

### CODE REVIEW

File: code/game/data_huds.dm [hunk 1]
```diff
@@ -127,4 +127,32 @@ proc/process_sec_hud(var/mob/M, var/advanced_mode,var/mob/eye)
 						holder.icon_state = "hudreleased"
 					else
 						continue
-				C.images += holder
\ No newline at end of file
+				C.images += holder
+
+mob/proc/handle_glasses_vision_updates(var/obj/item/clothing/glasses/G) //Unsure of where to put this, but since most of it is HUDs it seemed fitting.
```
Comment (PJB3005) [inline code/game/data_huds.dm]: `/` here.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

File: code/game/data_huds.dm (hunk 1 above)
Comment (9600bauds) [reply code/game/data_huds.dm]: /?

Comment (PJB3005) [reply code/game/data_huds.dm]: At the beginning of the line.

It's basically repo standard.

Comment (9600bauds) [reply code/game/data_huds.dm]: Oh right I completely missed that

**Review by PJB3005 — APPROVED** [review]

**Pull request merged**